### PR TITLE
Max ratio trade limits and tradeslip fix

### PR DESCRIPTION
**Pull request by TvrtkoM**
issues #251 #306

### CODE REVIEW

File: lib/constants.ts [hunk 1]
```diff
@@ -11,6 +12,8 @@ import {
 
 export const ZTG = 10 ** 10;
 
+export const MAX_IN_OUT_RATIO = new Decimal(1).div(3).toString();
```
Comment (Robiquet) [inline lib/constants.ts:15]: Did you consider get this from the chain config? Should be on the `swaps` property

File: lib/stores/TradeSlipStore.ts [hunk 2]
```diff
@@ -24,6 +28,7 @@ import {
   generateSwapExactAmountInTx,
   generateSwapExactAmountOutTx,
 } from "lib/util/pool";
+import { IoMdGitMerge } from "react-icons/io";
```
Comment (Robiquet) [inline lib/stores/TradeSlipStore.ts:31]: Don't think this is used

File: lib/constants.ts (hunk 1 above)
Comment (TvrtkoM) [reply lib/constants.ts:15]: Was planning to, but Malte said it's going to be removed from the chain config in near future so I proceeded like this.

Comment (Robiquet) [reply lib/constants.ts:15]: Ah good to know

**Review by Robiquet — APPROVED** [review]

**Pull request merged**